resource_booking/models/calendar_event.py

```diff
@@ -52,14 +52,28 @@ def write(self, vals):
         rescheduled._validate_booking_modifications()
         return result
 
+    def _notify_thread(self, message, msg_vals=False, **kwargs):
+        """We intercept this method because we can not overwrite the
+        _send_mail_to_attendees() method in calendar.attendee because it is done
+        with all the attendees and there we cannot indicate the mail_notify_author
+        context, if we did it, the email of each attendee would be sent additionally
+        to the author."""
+        if self.env.context.get("mail_notify_only_attendee_author"):
+            if msg_vals.get("author_id") in msg_vals.get("partner_ids"):
+                self = self.with_context(mail_notify_author=True)
```

Comment:
This problem **only** happens when creating a calendar event linked to `resource.booking` because we want it to notify all attendees (including the author of the email = organizer) but we do NOT want the organizer to receive all emails that are sent to all attendees.